{"name": "test_addgroup_failure_001", "status": "failed", "statusDetails": {"message": "AssertionError: 405 != 400", "trace": "self = <test_Addgroup.TestAddgroup testMethod=test_addgroup_failure_001>\n\n        def test_addgroup_failure_001(self):\n            response = requests.get('http://127.0.0.1:8000/home/api/groups/addgroup/', params={\n      \"name\": \"Python 学习小组\",\n      \"description\": \"这是一个学习 Python 的小组。\",\n      \"created_by\": 3\n    })\n>           self.assertEqual(response.status_code, 400)\nE           AssertionError: 405 != 400\n\ntestplatform\\tests\\unit\\test_Addgroup.py:23: AssertionError"}, "start": 1743341764817, "stop": 1743341764833, "uuid": "a09d5d58-b38a-4da3-be25-4f629eddb22a", "historyId": "6cabe8594f4e93ed56f76d536d29f8e1", "testCaseId": "6cabe8594f4e93ed56f76d536d29f8e1", "fullName": "testplatform.tests.unit.test_Addgroup.TestAddgroup#test_addgroup_failure_001", "labels": [{"name": "parentSuite", "value": "testplatform.tests.unit"}, {"name": "suite", "value": "test_Addgroup"}, {"name": "subSuite", "value": "TestAddgroup"}, {"name": "host", "value": "HO"}, {"name": "thread", "value": "17648-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testplatform.tests.unit.test_Addgroup"}]}